### ISSUE TITLE
Simple step-by-step

### ISSUE THREAD

**Issue by benlarsendk** [mitre/multiscanner#115]
Wouldn't it be a good idea to create a simple introduction to the system, i.e. how to go from cloning the repo, to actually be able to analyze a file. This would be a great part, such that more people can use the framework.

**Comment by awest1339** [comment]
That is a good point. If you would like to PR some, that would be much appreciated. Otherwise, we will add those as soon as possible.

Thanks!

**Comment by benlarsendk** [comment]
I wish I could do that, though I'm actually the guy who doesn't know how to make it working. ;-) 
I like how the whole thing works on paper, and I can really see some future in the project. Looking forward to see follow this. :+1:

**Comment by roswitina** [comment]
👍

**Comment by clenk** [comment]
The quickest way to get started is to use Docker as described [here](https://github.com/mitre/multiscanner/blob/master/docs/docker_standalone.md), at least for personal rather than production use. We should probably point that out in the README.

**Comment by benlarsendk** [comment]
@clenk yes, that would be a good idea. I found it reading the documentation. Though I still think some introductions how to set it up in the right way. E.g. if a person just wants to scan a file using the terminal with a couple of engines, or how you might set up the webservice yourself :)

**Comment by clenk** [comment]
@benlarsendk we just updated our readme and documentation. :) Do these changes help address this issue?

**Comment by benlarsendk** [comment]
Yes. This is great :) I'd like to keep this open, and I'll read the doc more carefully over the weekend. Of course you can close it if you want, it's your project.
I just think it's such a great setup you guys have made, that I really think the doc's should be foolproof, so anybody can use this masterpiece :)
